WebDevStudios/nextjs-wordpress-starter
# Downgrade tailwind css

Related to #1009

Closes N/A

### Description

The starter does not work with the latest version of tailwindcss. This is a temporary fix that downgrades the tailwind version while we properly fix it.

### Verification

How will a stakeholder test this?

1. Install the starter from scratch
2. Run `npm i` and `npm run build`
3. Verify that you do not see the error pointed out in https://github.com/WebDevStudios/nextjs-wordpress-starter/issues/1009
